src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import Image from "next/image";
+
+import * as S from "./styles";
+
+const ContactUsContent: React.FC = () => {
+  const [name, setName] = useState<string | null>(null);
+  const [subject, setSubject] = useState<string | null>(null);
+  const [content, setContent] = useState<string | null>(null);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (name === null) {
+      setName("");
+    }
+
+    if (subject === null) {
+      setSubject("");
+    }
+
+    if (content === null) {
+      setContent("");
+    }
+
+    if (!name || !subject || !content) {
+      return;
+    }
+
+    const body =
+      `Dear Bernard,\n\n` +
+      `${content}\n\n` +
+      `Thank you.\n\n` +
+      `Best regards,\n\n` +
+      `${name}\n`;
+
+    location.href =
```

Comment:
What is `location.href`, and how this relates to `mailto`

Reply:
location.href is a property and we can set it by location.href = URL. In  this case, we are using the mailto protocol as the URL such that we can use the email service.

Reply:
yup really nice!